libyasm/section.c

```diff
@@ -629,6 +629,11 @@ yasm_bytecode *
 yasm_section_bcs_append(yasm_section *sect, yasm_bytecode *bc)
 {
     if (bc) {
+    if (!sect || !bc) {
+        yasm_error_set(YASM_ERROR_VALUE, "Attempt to append bytecode to a NULL section or with a NULL bytecode");
+        return NULL;
+    }
+
```

Comment:
Needs indentation fixed to match following code.  It's also not possible for bc to be null here, since it's inside of a `if (bc)` block.

Reply:
changed

Reply:
Hi Dear [PeterJohnson](https://github.com/PeterJohnson),

Could you check the file I updated?

Thanks!